frontend/src/profiles/EditSkillList.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Chip from '@material-ui/core/Chip';
+import {makeStyles} from "@material-ui/core/styles";
+
+const useStyles = makeStyles(() => ({
+    skill: {
+        margin: "4px",
+        backgroundColor: "#d7385e"
+    }
+}));
+
+export default function EditSkillList({skills}) {
+
+    const classes = useStyles();
+
+    return(
+        <>
+            <h5>Skills:</h5>
```

Comment:
I remember that we talked about sizing with h1-5. Jan told us that we should try to do without it. But i can't remember what we should rather use :D